### ISSUE TITLE
when will the code be released?

### ISSUE THREAD

**Issue by Tord-Zhang** [richzhang/antialiased-cnns#1]
Hi, when will the code be publicly available？Thanks.

**Comment by richzhang** [comment]
I do not know for certain when, but am working to do so. Thank you very much for your interest and patience.

**Comment by edgarriba** [comment]
@richzhang in case you are interested, in [torchgeometry.image](https://torchgeometry.readthedocs.io/en/latest/filtering.html) we already have implemented some low-level vision stuff that can be integrated as backpropagable layers.

**Comment by richzhang** [comment]
Nice, thanks for the reference!

**Comment by minhmanho** [comment]
Simple but awesome. Thank you for your work. It would improve perceptual loss as well as generative task. Will you publish your trained models (well-trained with MaxBlurPool, etc)?

**Comment by richzhang** [comment]
Yes, that's the plan - apologies for the delay.

**Comment by shivamsaboo17** [comment]
Great paper! Given, it improves the performance of image translation do you think it can help in making video object segmentation more temporally consistent or less flickering? Thanks in advance.

**Comment by richzhang** [comment]
Thanks -- yes, I haven't tried it, but I think it has a good chance to. Let me know if you try it.

**Comment by richzhang** [comment]
The code has been released under this repository: https://github.com/adobe/antialiased-cnns.

Thanks everyone for your patience.
